### ISSUE TITLE
syncoid to non-zfs backup system

### ISSUE THREAD

**Issue by stuartthebruce** [jimsalterjrs/sanoid#564]
It may be out of scope for syncoid, but it would be nice if it could send zfs streams to a non-zfs system to backup incremental snapshot streams as binary blobs, e.g., a tape backup system.

Two quick ideas are:

1. require a small zfs bookkeeping instance on the destination system that does not hold any data but creates empty shadow snapshots of all successfully transferred streams. However, this would either require syncoid to match snapshot guid and timestamps between the source and destination as they would no longer be identical, or require a modified version of zstreamdump to pass a stripped down version of the stream to zfs recv that does not have vdev data changes.
2. Keep track of the state of remote snapshot streams elsewhere than zfs on the target. For example, zfs on the source with custom zfs properties, or /bin/ls, /bin/stat, and/or a text file on a remote Posix filesystem.

Thanks.

**Comment by secabeen** [comment]
This is an interesting idea, but is pretty fragile, as ZFS streams are not redundant in any way.  Any damage to them at all is likely to cause a complete failure to import.  If syncoid was going to this, it should also calculate and save a PAR2 file for each stream, to ensure durability over time.

**Comment by stuartthebruce** [comment]
Excellent idea. I was unaware of PAR2, but will definitely give that a try in addition the builtin filesystem level checksumming I am considering using (VSM).

**Comment by phreaker0** [comment]
duplicate of #35 #465
tldr: this won't be included in syncoid
